src/test/java/guitests/UIComponentsTest.java

```diff
@@ -25,20 +25,8 @@ public void keywordCompletionTest() {
     @Test
     public void filterTextFieldTest() {
         doubleClick("#dummy/dummy_col0_filterTextField");
-        click("#dummy/dummy_col0_filterTextField");
         push(KeyCode.BACK_SPACE);
-        type("is");
-        push(KeyCode.ESCAPE);
-        press(KeyCode.SHIFT).press(KeyCode.DIGIT9).release(KeyCode.DIGIT9).release(KeyCode.SHIFT);
-        type("is");
-        press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
-        push(KeyCode.LEFT);
-        push(KeyCode.TAB);
-        push(KeyCode.RIGHT);
-        type("open OR is");
-        press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
-        type("closed");
-        press(KeyCode.SHIFT).press(KeyCode.DIGIT0).release(KeyCode.DIGIT0).release(KeyCode.SHIFT);
+        type("(is:open OR is:closed)");
```

Comment:
Hmm, but if it failed to type parentheses there, the test would still pass. Is there some other way we could test the method?

Maybe a new test: start up the GUI, have it type a bunch of special characters into the filter box (doesn't have to be valid syntax), then read the value of the field and assert that everything expected is there?